aten/src/ATen/native/cuda/Math.cuh

```diff
@@ -299,14 +407,100 @@ static inline C10_HOST_DEVICE scalar_t calc_i0e(scalar_t _x) {
   accscalar_t x = ::abs(static_cast<accscalar_t>(_x));
 
   if (x <= 8.0) {
-    const auto A = chebyshev_coefficients_A<accscalar_t>();
+    const auto A = chebyshev_coefficients_i0e_A<accscalar_t>();
     accscalar_t y = static_cast<accscalar_t>((x / 2.0) - 2.0);
     return static_cast<scalar_t>(chbevl(y, A, 30));
   }
 
-  const auto B = chebyshev_coefficients_B<accscalar_t>();
+  const auto B = chebyshev_coefficients_i0e_B<accscalar_t>();
   return static_cast<scalar_t>(chbevl(static_cast<accscalar_t>(32.0 / x - 2.0), B, 25) / ::sqrt(x));
 }
 
+template <typename scalar_t>
+static inline C10_HOST_DEVICE scalar_t calc_i1(scalar_t _x) {
```

Comment:
So bfloat16 and float16 would call this function, and then be upcast to float, but this is not using the float-specific values?

Reply:
Killed this part of the code. Kernel is note instantiated for float16 and bfloat16.